DesignSafe-CI/portal
# Task/DES-558: get rid of djng

## Overview: ##
![image](https://user-images.githubusercontent.com/12601812/125121458-95b26480-e0b9-11eb-8ce0-c905653c3cad.png)

## PR Status: ##


## Related Jira tickets: ##

* [DES-538](https://jira.tacc.utexas.edu/browse/DES-538)

## Summary of Changes: ##
- Replace `djangoUrl.reverse()` calls with correct URL paths. 
- Delete the vendored djng library
- Uninstall djng and its middleware in the django backend.

## Testing Steps: ##
1. Click through and confirm that there are no import/injection errors in the console. The areas affected most by this PR are the Workspace and Notifications.

## UI Photos:

## Notes: ##
